crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4087,19 +4071,41 @@ pub struct AdyenPayoutCancelRequest {
     merchant_account: Secret<String>,
 }
 
+#[cfg(feature = "payouts")]
+impl TryFrom<&PayoutMethodData> for PayoutCardDetails {
+    type Error = Error;
+    fn try_from(item: &PayoutMethodData) -> Result<Self, Self::Error> {
+        match item {
+            PayoutMethodData::Card(card) => Ok(Self {
+                _type: "scheme".to_string(), // FIXME: Remove hardcoding
```

Comment:
We should avoid having underscores in identifier names since Rust interprets these differently (unused). If this is for JSON (de)serialization support, you can name the field `card_type` and annotate it with `#[serde(rename = "_type")]` to use `_type` for (de)serialization